scripts/o3de/o3de/export_project.py

```diff
@@ -49,6 +50,17 @@ def args(self) -> list:
         return self._args
 
 # Helper API
+def determine_host_asset_platform():
```

Comment:
why have this mapping? Iinstruct users to use "windows", "linux", or "darwin" when providing the exporting platform, and keep the O3D standard :)

Reply:
Currently that is used to determine a default asset platform on the user's machine if no argument is supplied to the export script.
```
parser.add_argument('-pl', '--platform', type=str, default=host_asset_platform, choices=['pc', 'linux', 'mac'])
```
So if that argument is not supplied, the assumption is that the user wants to build it on their own machine. Doing it this way would make it cross platform for each host that o3de supports (the technique was suggested by @lumberyard-employee-dm ). As for O3DE standard, if I understand correctly, `[pc, linux, mac]` are currently the o3de standard internally.

Reply:
Maybe it'd be clearer if you rename the function to be something like `get_default_asset_pllatform`